guides/source/command_line.md

```diff
@@ -413,7 +413,7 @@ Any modifications you make will be rolled back on exit
 irb(main):001:0>
 ```
 
-#### The app and helper objects
+#### The App and Helper Objects
```

Comment:
Here, "app" and "helper" refer to the `app` and `helper` values, so it may be better to wrap in backticks:

```suggestion
#### The `app` and `helper` Objects
```